candle-nn/Cargo.toml

```diff
@@ -33,7 +33,8 @@ criterion = { workspace = true }
 default = []
 accelerate = ["dep:accelerate-src", "candle/accelerate"]
 cuda = ["candle/cuda"]
-mkl = ["dep:intel-mkl-src", "candle/mkl"]
+_mkl = ["dep:intel-mkl-src", "candle/_mkl"]
+mkl = ["candle/mkl"]
```

Comment:
Isn't this different from the previous behavior where when `mkl` is enabled we would have a proper dependency to `intel-mkl-src` here (with the "mkl-static-lp64-iomp" feature)?